Cargo.toml

```diff
@@ -20,6 +20,7 @@ crc32fast = "1.4.2"
 crossbeam-channel = "0.5.13"
 crossbeam-skiplist = "0.1.3"
 dotenvy = { version = "0.15.7" }
+env_logger = { version = "0.11.5", optional = true }
```

Comment:
I removed this and replaced it with tracing's logger. It can output to a variety of loggers including env logger. See the main in bencher on how to init it.

Reply:
ah! thanks for catching my newbie error, I was very confused why I couldn't find any existing logging library by looking for the word "log" in the Cargo.toml

Reply:
Yea, the logging is confusing to me, too. I'm still not 100% sure I have it right. It's not totally clear to me whether we should be using `tracing::warn!` or `log::warn!`..